test/utils.py

```diff
@@ -934,3 +944,34 @@ def run_with_config(self, config):
             with self.ini_path.open("w") as f:
                 f.write(config_old)
             self.admin("RELOAD")
+
+
+class OpenLDAP:
+    def __init__(self, config_dir):
+        self.port_lock = PortLock()
+        self.config_dir = config_dir
+        self.slapd_pid_file = self.config_dir / "ldap" / "slapd.pid"
+
+    def startup(self):
+        print(
+            subprocess.run(
```

Comment:
We have a nicer `run` helper to run commands.
```suggestion
            run(
```